pkg/internal/cluster/create/actions/config/config.go

```diff
@@ -85,12 +84,28 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 	if err != nil {
 		return err
 	}
+
+	kubeadmConfigPlusPatches := func(node nodes.Node, configData kubeadm.ConfigData) func() error {
+		return func() error {
+			// determine if we're dealing with a worker node
+			// so we can include it's corresponding patches
+			includeWorkerPatches := !configData.ControlPlane
```

Comment:
Why are worker nodes being special cased?
Why not just include any node level patches?

Reply:
The idea was to only apply the worker patches to workers and not to the control planes nodes, but this should also merge with the cluster level patches.